src/modules/displayComment.js

```diff
@@ -0,0 +1,32 @@
+import getComment from './getComment.js';
+
+let counter = 0;
+const displayComment = async (id) => {
+  const commentData = await getComment(id);
+  counter = commentData.length;
+  const totalCount = document.querySelector('.number');
+  const comments = document.querySelector('.comment-holder');
+  const shwComments = document.querySelector('.comments');
+
+  commentData.forEach((com) => {
+    shwComments.innerHTML += `
+        <p class="addedComment"><b>UserName</b>: ${com.username}</br></br> <b>Comment</b>: </br> ${com.comment}</p></br><hr/>
+    `;
+  });
+  comments.appendChild(shwComments);
+  totalCount.textContent = counter;
+};
```

Comment:
- [ ] Like the items counter, the comments counter should be a function implemented in a separate module where you can import from different files like the test file. Kindly extract the function that counts the comments and implement it in a new module.